jsapp/js/components/accountSettings.es6

```diff
@@ -143,7 +144,7 @@ export var AccountSettings = React.createClass({
     var accountName = stores.session.currentAccount.username;
 
     return (
-      <DocumentTitle title={`${accountName} | KoboToolbox`}>
+      <DocumentTitle title={`${accountName} | ${ONA_TITLE}`}>
```

Comment:
why change the var style? Should this stay consistent with everything else and use camel case, so `onaTitle`